Resources/Prototypes/Entities/buildings/storage/crate_types.yml

```diff
@@ -118,3 +118,163 @@
 
   - type: Icon
     state: o2crate
+
+- type: entity
+  id: crate_flashlight
+  name: Flashlight Crate (x5)
+  parent: crate_generic
+  components:
+    - type: StorageFill
+      contents:
+        - Flashlight
+        - Flashlight
+        - Flashlight
+        - Flashlight
+        - Flashlight
+
+- type: entity
+  id: crate_light_bulb
+  name: Light Bulb Crate (x10)
+  parent: crate_generic
+  components:
+    - type: StorageFill
+      contents:
+        - Light Bulb
```

Comment:
Does this even work? The id for lightbulbs is `LightBulb` not `Light Bulb`
Ditto for any others.

Reply:
I just tested it, and it appears to not work.

Reply:
But then, that might also be because this particular branch is so old that `StorageFill` appears to not be a component.

Reply:
where isn't `StorageFill` a component? CC's branch or master?

Reply:
On the PR branch. It's defined on master.

Reply:
Ah, I was going to say!